### PR TITLE
Email confirm fix v2.0

### PR DESCRIPTION
**Pull request by kevinNghiem**

### CODE REVIEW

File: src/pages/ConfirmationPage.js [hunk 1]
```diff
@@ -54,8 +56,10 @@ const EmailError = () => {
 class ConfirmationPage extends React.Component {
   constructor(props) {
     super(props)
-    this.state = { sending: false }
+    this.state = { sending: false, comfirmNum: [] }
     this.handleSubmit = this.handleSubmit.bind(this)
+    // eslint-disable-next-line no-console
```
Comment (andr0272) [inline src/pages/ConfirmationPage.js]: Theres no console log here

File: src/pages/ConfirmationPage.js [hunk 2]
```diff
@@ -70,20 +74,20 @@ class ConfirmationPage extends React.Component {
...
-    for (i = 0; i < keys.length; i++) {
-      chr = keys.charCodeAt(i)
-      hash = (hash << 5) - hash + chr
-      hash |= 0
-    }
-    return hash
+  getEmailNum(documentId) {
+    axios
+      .get(`/appointments/confirm/${documentId}`)
+      .then(locs => {
```
Comment (andr0272) [inline src/pages/ConfirmationPage.js]: This var shouldnt be called `locs` if its pulling appointment data

File: src/email/handleSubmitEmail.js [hunk 3]
```diff
@@ -1,42 +1,73 @@
...
+  http
+    .get(`${apiHost}/appointments/confirm/${documentId}`, resp => {
+      logDebug(`STATUS: ${resp.statusCode}`)
+      logDebug(`HEADERS: ${JSON.stringify(resp.headers)}`)
+      resp.on('data', chunk => {
+        data += chunk
+      })
+      resp.on('end', function() {
+        data = JSON.parse(data)
+        // eslint-disable-next-line no-console
```
Comment (EricDube) [inline src/email/handleSubmitEmail.js]: Remove this, unneeded

File: src/pages/ConfirmationPage.js (hunk 2 above)
Comment (kevinNghiem) [reply src/pages/ConfirmationPage.js]: fixed

File: src/email/handleSubmitEmail.js (hunk 3 above)
Comment (kevinNghiem) [reply src/email/handleSubmitEmail.js]: fixed

File: src/email/handleSubmitEmail.js [hunk 4]
```diff
@@ -1,42 +1,72 @@
-import { sendNotification } from './sendmail';
-import { logerror } from '../utils/logger';
+import { sendNotification } from './sendmail'
+import { logDebug, logError } from '../utils/logger'
+import http from 'http'
 
+const apiHost = process.env.CONNECTION_STRING
 const CONFIRMATION_EMAIL = 'af563da5-43bd-4b9a-9610-e3990a7f4315'
```
Comment (will0684) [inline src/email/handleSubmitEmail.js]: Is this our API key? Shouldn't we store this in an environment variable? 🤔

**Pull request closed without merging**